Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -286,7 +286,6 @@ static MTLStoreAction storeAction(WGPUStoreOp storeOp)
             mtlAttachment.resolveLevel = 0;
             mtlAttachment.resolveSlice = 0;
             mtlAttachment.resolveDepthPlane = 0;
-            mtlAttachment.storeAction = MTLStoreActionMultisampleResolve;
```

Comment:
see line 282 were we set the storeAction to `MTLStoreActionStoreAndMultisampleResolve` which is needed for these samples